src/tfms-all.props

```diff
@@ -0,0 +1,16 @@
+<Project>
```

Comment:
I'm not sure to understand what the difference between "all" and "winui" / "uwp". please add comment to explain what's are the targets and when we should use this instead of the "winui"/ "uwp".

Also as this targets "UI platforms", I would somehow add "ui" in the filename. "all" seams a bit too generic.

Reply:
@dr1rrb I've updated file names and added comments in the props files - let me know if these make sense